src/Programs/Programs.ts

```diff
@@ -255,9 +280,9 @@ export const Programs: Record<CompletedProgramName, Program> = {
       time: CONSTANTS.MillisecondsPerQuarterHour,
     },
     run: (): void => {
-      Terminal.print("This executable cannot be run.");
-      Terminal.print("AutoLink.exe lets you automatically connect to other servers when using 'scan-analyze'.");
-      Terminal.print("When using scan-analyze, click on a server's hostname to connect to it.");
+      Terminal.error("This executable cannot be run.");
+      Terminal.warn("AutoLink.exe lets you automatically connect to other servers when using 'scan-analyze'.");
```

Comment:
I'm going to comment on all of the following simply with "info" to say that these are informational messages only so I don't think it's appropriate to use .warn